### ISSUE TITLE
2018 kick off - fixing all severity 1 and 2 bugs

### ISSUE THREAD

**Issue by daniellemoorhead** [openfoodfoundation/openfoodnetwork#2065]
## Description
As part of the defined 2018 roadmap the biggest priority we have in the "Make what we have great!" goal is to identify all bugs that are severity 1 or 2, and then fix these.

This epic is the parent for those current bugs that need to be fixed. Once resolved, this epic can be moved to done, and subsequent severity 1 and 2 bugs can be handled individually within the delivery pipe process.

**Comment by myriamboure** [comment]
Yep @daniellemoorhead ! I'm reviewing all the bug to assign them severity, we have another session with Enrico next week, so when we have reviewed them all, we can just put the ones that are not started and well specified in "dev ready", if not really well specified in "delivery train backlog", and just close this epic. I guess when we have cleaned up the existing, when there is a new one we will treat it immediately (decide if severity is well set and put in backlog or dev ready) so we won't need that epic anymore.

**Comment by myriamboure** [comment]
@daniellemoorhead I'm closing this epic now that we've gone through all the s1 and s2 issues with @enricostano . The plan from now is that each time someone mark a new issue as s1 or s2 we decide straight away if we agree on severity and if yes but it directly in either delivery backlog is ready to be taken or blocked column if this is an important bug but blocked by something (we added that column in delivery board so that we don't forget about the important blocked things...). Hope that sounds good to you, if not raise your voice!

**Comment by daniellemoorhead** [comment]
Perfect! I've moved the epic into the review column on the product feature backlog. We haven't really discussed how we would use this column, but it seems like a good point in time to reflect on what we did and take away some lessons learned?

**Comment by myriamboure** [comment]
For me the "review" column was supposed to check if what we have done has solved the problem in a satisfying way or not. Or if we need to put it  back in inception to iterate again and move forward on the problem resolution.  For this one I think the process we have set up is solving our problem that we don't have a process to filter and push bugs into the delivery pipeline, so I would say it's good enough and we can just close the epic :-)

**Comment by enricostano** [comment]
We can close this epic, but I think it is being super useful the bi-weekly bugs triage meeting. Please let's keep doing it :pray:

**Comment by myriamboure** [comment]
Sure, let's keep doing that @enricostano :-)

**Comment by daniellemoorhead** [comment]
@myriamboure and @enricostano you guys are the best!
